src/stan/math/prim/scal/prob/neg_binomial_2_rng.hpp

```diff
@@ -31,17 +31,44 @@ namespace stan {
                        RNG& rng) {
       using boost::variate_generator;
       using boost::random::negative_binomial_distribution;
+      using boost::random::poisson_distribution;
+      using boost::gamma_distribution;
 
       static const char* function("stan::math::neg_binomial_2_rng");
+      static const double POISSON_MAX_RATE = std::pow(2.0, 30);
```

Comment:
We should move `POISSON_MAX_RATE` to `src/stan/math/prim/scal/fun/constants.hpp`. Right now, it's defined in 3 places.